src/Components/About/AboutUsMembersList.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
```

Comment:
maybe the name of this component should be AboutUsMembersItem and not list because this is not the list, is the item of the list

Reply:
You're right